x-pack/plugins/data_visualizer/public/application/index_data_visualizer/search_strategy/requests/get_data_view_by_index_pattern.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { DataView, DataViewsContract } from '@kbn/data-views-plugin/public';
+
+export async function getDataViewByIndexPattern(
```

Comment:
there is more going on than just getting a date view based on the index pattern, e.g. creating of the data view. 
I reckon it's worth renaming a providing a comment.